blocks/footer/footer.css

```diff
@@ -1,125 +1,146 @@
 footer {
   background-color: #222;
   color: var(--c-primary-white);
+  position: relative;
+  padding: 0 20px;
 }
 
-.footer .section {
-  padding: 40px 15px 0;
-}
-
-.footer .section:last-child {
-  padding-bottom: 40px;
-}
-
-.footer .footer-logo img {
-  width: 250px;
-  height: auto;
+.footer .columns-container .columns-wrapper {
+  padding-top: 40px;
 }
 
 .footer .footer-column > * {
```

Comment:
Can you combine this and the next rule at 16

Reply:
great catch!